Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentMode.h

```diff
@@ -94,12 +100,23 @@ namespace WhiteBox
         SubMode m_currentSubMode = SubMode::Default;
         bool m_restoreModifierHeld = false;
 
+        AzToolsFramework::ViewportUi::ClusterId
+            m_transformClusterId; 
+        AzToolsFramework::ViewportUi::ButtonId
+            m_transformTranslateButtonId; 
+        AzToolsFramework::ViewportUi::ButtonId
+            m_transformRotateButtonId; 
+        AzToolsFramework::ViewportUi::ButtonId
+            m_transformScaleButtonId; 
```

Comment:
I've format these normally so they're all on the same line (and if they need a comment use `//!` on the line above as opposed to `//!<` on the formatting for line length looks a bit strange)